From ec12c163aab22fa08d6797d182b7e296cead0c10 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Fri, 21 Feb 2020 19:12:32 +0530
Subject: [PATCH] meson: host_system is ios inside cerbero

This is actually the only way right now to differentiate between iOS
and macOS inside Meson, but sadly it's not entirely standardized
upstream: https://mesonbuild.com/Reference-tables.html#operating-system-names

We can upstream this to glib when things get standardized in Meson:
https://github.com/mesonbuild/meson/issues/6257
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 484f4c750..00b3bda32 100644
--- a/meson.build
+++ b/meson.build
@@ -2026,7 +2026,7 @@ if host_system == 'windows'
   export_dynamic_ldflags = []
 elif host_system == 'cygwin'
   export_dynamic_ldflags = ['-Wl,--export-all-symbols']
-elif host_system == 'darwin'
+elif host_system in ['darwin', 'ios']
   export_dynamic_ldflags = []
 elif host_system == 'sunos'
   export_dynamic_ldflags = []
-- 
2.24.1

